README.md

```diff
@@ -7,23 +7,24 @@ Zowe Explorer is a separately delivered component of Zowe belonging to the [Zowe
 
 The Zowe Explorer extension modernizes the way developers and system administrators interact with z/OS mainframes. Working with data sets and USS files from VSC can be more convenient than using 3270 emulators, and complements your Zowe CLI experience. The extension provides the following benefits:
 
-* Enables developers to create, modify, rename, copy and upload data sets directly to a z/OS mainframe.
-* Enables developers to create, modify, rename and upload USS files directly to a z/OS mainframe.
+* Enables you to create, modify, rename, copy, and upload data sets directly to a z/OS mainframe.
+* Enables you to create, modify, rename, and upload USS files directly to a z/OS mainframe.
 * Provides a more streamlined way to access data sets, uss files and jobs.
-* Lets you create and use Zowe CLI `zosmf` compatible profiles.
+* Lets you create, edit, and delete Zowe CLI `zosmf` compatible profiles.
+* Lets you use the Secure Credential Store plug-in to store your credentials securely in the settings.
 
 **Note:** The Zowe Explorer is powered by [Zowe CLI](https://zowe.org/home/). The extension demonstrates the potential for plug-ins powered by Zowe.
 
 **Tips:**
 
 * For information about how to install the extension from a `VSIX` file and run system tests on the extension, see the [Developer README](https://github.com/zowe/vscode-extension-for-zowe/blob/master/docs/README.md).
 * For information about how to develop for Eclipse Theia, see [Theia README](https://github.com/zowe/vscode-extension-for-zowe/blob/master/docs/README-Theia.md).
-* For information about how to create a VSCode extension for Zowe Explorer (a Zowe Explorer extension) see [VSCode extensions for Zowe Explorer](https://github.com/zowe/vscode-extension-for-zowe/blob/master/docs/README-Extending.md).
```

Comment:
Hey @IgorCATech  and @Colin-Stone,
Should this be added back?

Reply:
yes, apparently it was added after I copied the md file and worked on it locally, and then just copied the entire file with changes back. Sorry, my bad.
Good catch, @zFernand0 !